### ISSUE TITLE
shellcheck config not used

### ISSUE THREAD

**Issue by altendky** [rhysd/actionlint#486]
When actionlint is extracting a `run:` string from a workflow file the shellcheck config is not respected.  I'm guessing that either the string is piped directly into shellcheck or located in a tmp directory outside of the project structure.  Normally I would consider this good but since shellcheck (afaik) looks for the config based on the location of the script being analyzed, this could be making it not find the project config.

**Comment by rhysd** [comment]
Yes. The config file is intentionally not referred (`--norc` option passed to `shellcheck`). Instead you can use `SHELLCHECK_OPT` to pass options to shellcheck process.

https://github.com/rhysd/actionlint/blob/main/docs/checks.md#check-shellcheck-integ

**Comment by altendky** [comment]
I am interested in using the same config for all shell scripts, inside a workflow or as a standalone file.  Certainly, in some cases different configurations would be wanted, but defaulting to the same seems useful.

As far as any potential implementation, I could imagine an argument that shellcheck should first grow a feature like `--search-for-rc-from` and you can pass a file or directory to use for the config search instead of the file passed for checking.  I don't know if they would argue against this or not.
